src/components/Gesture.svelte

```diff
@@ -271,38 +271,37 @@
 </script>
 
 <!-- Recording countdown popup -->
-<StandardDialog isOpen={showCountdown || isThisRecording} onClose={cancelRecording}>
-  <div class="flex flex-col items-center gap-8 w-120">
-    <h2 class="text-xl font-bold self-start">
-      {$t('content.data.recordingDialog.title', { values: { action: $nameBind } })}
-    </h2>
-    <div class="flex flex-col space-y-3 self-center items-center justify-center">
-      <div class="flex justify-center">
-        <p class="text-lg px-10 text-center">
-          {$t('content.data.recording.description')}
+<MeltDialog
+  isOpen={showCountdown || isThisRecording}
+  onClose={cancelRecording}
+  class="flex flex-col items-center gap-8 w-120"
+  headingClass="text-xl font-bold self-start"
+  heading={$t('content.data.recordingDialog.title', { values: { action: $nameBind } })}>
```

Comment:
I used `heading` instead of `headingId` as prop for MeltDialog due to this usage where the translation string might need values. I also added a `headingClass` prop for MeltDialog because of this use case where the heading styling is different